nimbus/src/nimbus.idl

```diff
@@ -19,8 +19,14 @@ dictionary EnrolledExperiment {
     string user_facing_description;
     string branch_slug;
     string enrollment_id;
+    sequence<Branch> branches;
 };
 
+dictionary Branch {
+    string slug;
+    u32 ratio;
```

Comment:
This won't work without the missing `feature` definition.  It has to match the one defined in lib.rs exactly, which means you will also need to add the `FeatureConfig` dictionary definition to the IDL.